app/views/projects/_project-card.html.erb

```diff
@@ -2,24 +2,25 @@
   <div class="w-full bg-center bg-cover" style="height: 130px; background-image: url(<%= project.cover_photo(local_assigns[:group]) %>);"></div>
 
   <% if project.accepting_volunteers? %>
-    <div class="absolute inset-auto right-0 top-0 mr-4 mt-4 z-10 text-white rounded-full px-2 py-1 text-sm" style="background-color: rgba(0, 0, 0, 0.5);"><%= inline_svg_pack_tag 'media/svgs/status-bolt.svg', class: 'inline' %>Actively Recruiting</div>
+    <div class="absolute inset-auto right-0 top-0 mr-4 mt-4 z-10 text-white rounded-full px-3 py-1 text-xs" style="background-color: rgba(0, 0, 0, 0.5);"><span class="pr-1"><%= inline_svg_pack_tag 'media/svgs/status-bolt.svg', class: 'inline' %></span> Actively Recruiting</div>
   <% end %>
 
-  <div class="p-4">
-    <div class="text-2xl truncate leading-tight mb-1"><%= project.name %></div>
-    <div class="text-gray-500 text-sm mb-2 leading-tight overflow-hidden" style="height: 50px;"><%= project.description.truncate 150 %></div>
+  <div class="px-5 pt-5 pb-6">
```

Comment:
The padding is a bit too much for my taste. I would use a `.p-4` all-round, but it's up to you.